lib/fizzy/cxx20/numeric.hpp

```diff
@@ -0,0 +1,37 @@
+// Fizzy: A fast WebAssembly interpreter
+// Copyright 2019-2020 The Fizzy Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#if __cplusplus > 201703L
+
+#include <bit>
+
+namespace fizzy
+{
+template <class T>
+constexpr int popcount(T x) noexcept
```

Comment:
This is not needed and probably not used currently.

Reply:
What do you mean?